pkg/testutil/leak.go

```diff
@@ -106,6 +106,7 @@ func interestingGoroutines() (gs []string) {
 		}
 		stack := strings.TrimSpace(sl[1])
 		if stack == "" ||
+			strings.Contains(stack, "runtime/sigqueue.go") ||
```

Comment:
I think the leak go routine is from `signal_recv(...)`. that's why i whitelist the `sigqueue.go`
```1 instances of:
os/signal.signal_recv(...)
 /usr/local/Cellar/go/1.7.3/libexec/src/runtime/sigqueue.go:116 +0x157
os/signal.loop()
 /usr/local/Cellar/go/1.7.3/libexec/src/os/signal/signal_unix.go:22 +0x22
created by os/signal.init.1
 /usr/local/Cellar/go/1.7.3/libexec/src/os/signal/signal_unix.go:28 +0x41
exit status 1
```